skbio/io/format/blast6.py

```diff
@@ -0,0 +1,287 @@
+|qcovs      |Query Coverage Per Subject          |int  |
++-----------+------------------------------------+-----+
+|qcovhsp    |Query Coverage Per HSP              |int  |
++-----------+------------------------------------+-----+
+
+List of data and descriptions taken from BLAST Command Line Applications User
+Manual [2]_.
+
+Examples
+--------
+An example of using _blast6_to_data_frame with default column labels.
+
+>>> from io import StringIO
+>>> from skbio import read
+>>> import pandas as pd
+>>> fs = '\\n'.join([
+...     'query1\\tsubject1\\t30.0\\t3\\t5\\t1\\t1\\t3\\t8\\t7\\t0.030\\t13.2',
+...     'query2\\tsubject2\\t40.0\\t5\\t4\\t2\\t1\\t6\\t9\\t0\\t0.045\\t15.0'
+... ])
+>>> fh = StringIO(fs)
+
+Load BLAST+6 data into a DataFrame and specify that default columns should be
+used.
+
+>>> df = read(fh, format="blast+6", into=pd.DataFrame, default_columns=True)
+>>> df # doctest: +NORMALIZE_WHITESPACE
+   qseqid    sseqid  pident  length  mismatch  gapopen  qstart  qend  sstart \\
+0  query1  subject1      30       3         5        1       1     3       8
+1  query2  subject2      40       5         4        2       1     6       9
+<BLANKLINE>
+   send  evalue  bitscore
+0     7   0.030      13.2
+1     0   0.045      15.0
+
+An example of using _blast6_to_data_frame with custom column labels.
+
+>>> from io import StringIO
+>>> from skbio import read
+>>> import pandas as pd
+>>> fs = '\\n'.join([
+...     'subject2\\tquery1\\t100\\tN/A\\t0\\tsubject2\\t32\\t100.0',
+...     'subject1\\tquery2\\t70\\tN/A\\t0\\tsubject1\\t19\\t70.0',
+...     'subject2\\tquery1\\t100\\tN/A\\t0\\tsubject2\\t18\\t50.0'
+... ])
+>>> fh = StringIO(fs)
+
+Load BLAST+6 data into a DataFrame and specify your column types.
+
+>>> df = read(fh, format="blast+6", into=pd.DataFrame,
+...           columns=['sacc', 'qaccver', 'qcovs', 'sblastnames', 'gapopen',
+...                    'sallacc', 'score', 'ppos'])
+>>> df # doctest: +NORMALIZE_WHITESPACE
+       sacc qaccver  qcovs  sblastnames  gapopen   sallacc  score  ppos
+0  subject2  query1    100          NaN        0  subject2     32   100
+1  subject1  query2     70          NaN        0  subject1     19    70
+2  subject2  query1    100          NaN        0  subject2     18    50
+
+References
+----------
+.. [1] http://blastedbio.blogspot.com/2014/11/column-headers-in-blast-tabular-\
+and-csv.html
+.. [2] http://www.ncbi.nlm.nih.gov/books/NBK279675/
+"""
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from skbio.io import create_format
+import pandas as pd
```

Comment:
pandas and numpy imports (i.e. 3rd party dependencies) go in their own section before skbio imports